### PR TITLE
Meaningful PR name

### PR DESCRIPTION
**Pull request by Akceptor**
- Changes descriptions

### CODE REVIEW

File: src/main/java/converter/Converter.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import converter.exceptions.InvalidAmountOrExchangeRateException;
+import converter.interfaces.IRateProvider;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class Converter {
+	private final IRateProvider rateProvider;
+	private int scale = 2;
+	private final BigDecimal ONE_HUNDRED = BigDecimal.TEN.pow(2); // 100 = 10 ^ 2
```
Comment (Akceptor) [inline src/main/java/converter/Converter.java]: Оверінжинірінг, ні?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
+<!--
 # Converter
-Task:
+-->
```
Comment (Akceptor) [inline README.md:3]: I'd stick with markdown here, no real need for html IMO

File: src/main/java/converter/ConstantRateProvider.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.math.BigDecimal;
+
+/**
+ * There are no requirements for RateProvider implementation
+ * but interfaces without the implementation are considered a code smell sometimes
+ */
+public class ConstantRateProvider implements IRateProvider {
+    @Override
+    public BigDecimal getRateUsdToEur() {
```
Comment (Akceptor) [inline src/main/java/converter/ConstantRateProvider.java:27]: I don't think soma APIs will return exchange rate as BigDecimal, but let's consider this later :)

File: src/main/java/converter/Converter.java [hunk 4]
```diff
@@ -26,8 +26,8 @@ public class Converter {
 	 *
 	 * @param rateProvider	set a IRateProvider data type.
 	 * @param scale			the int data type (min = '0', max = '2E31 - 1' '').
-	 * @see					IRateProvider interface.
-	 * @see					BigDecimal class, method - setScale(int newScale, RoundingMode roundingMode).
```
Comment (Akceptor) [inline src/main/java/converter/Converter.java]: https://www.baeldung.com/java-method-in-javadoc

**Comment by AndrijVarenko** [comment]
Pull request JavaDoc code with main (first version).

**Pull request merged**